apollo-compiler/build.gradle.kts

```diff
@@ -30,7 +30,7 @@ abstract class GeneratePluginVersion : DefaultTask() {
 
   @org.gradle.api.tasks.TaskAction
   fun taskAction() {
-    val versionFile = File(outputDir.asFile.get(), "Version.kt")
+    val versionFile = File(outputDir.asFile.get(), "com/apollographql/apollo3/compiler/Version.kt")
```

Comment:
That change is not related to publishing. It makes sure the generated file is under a correct folder in the sourcesJar else we end up with `Version.kt` at the root of the Jar